src/tests/ftest/erasurecode/restart.py

```diff
@@ -54,8 +54,10 @@ def execution(self, agg_check=None):
         self.get_dmg_command().system_start()
 
         if agg_check == "After":
+            size_after_restart = self.pool.pool_percentage_used()
+            self.log.info("Size after Restarti: %s ", self.pool.pool_percentage_used())
             # Verify if Aggregation is getting started
-            if not any(check_aggregation_status(self.pool, attempt=50).values()):
+            if not size_after_restart['scm'] > size_before_restart['scm']:
```

Comment:
Seeing as the test never displays `size_before_restart` it would be useful for test failure triage to include the values that triggered the test failure to be included in the test failure message.

Reply:
done